learningresources/search_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+from haystack.indexes \
+    import SearchIndex, Indexable, CharField  # pragma: no cover
+
+from learningresources.models import LearningResource  # pragma: no cover
+
+
+class LearningResourceIndex(SearchIndex, Indexable):  # pragma: no cover
+    """
+    Index configuration for the LearningResource model.
+    """
+    # This is a config file for Haystack; nothing to test.
```

Comment:
Yep, you're right. I had a `skip` line in there which has since been removed, and the comment was to cover it. I'll fix that.